### PR TITLE
Update panos.md

### PR DESCRIPTION
**Pull request by yaks77**
Typo in the sourcetype asociated with the key pan_panos_globalprotect.  correct sourcetype is pan:globalprotect instead of pan:pan_globalprotect

### CODE REVIEW

**Comment by yaks77** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by yaks77** [comment]
I have read the Code of Conduct and I hereby accept the Terms

File: docs/sources/vendor/PaloaltoNetworks/panos.md [hunk 1]
```diff
@@ -32,7 +32,7 @@
 | key                     | sourcetype     | index          | notes          |
 |-------------------------|----------------|----------------|----------------|
 | pan_panos_log           | pan:log       | netops          | none          |
-| pan_panos_globalprotect | pan:pan_globalprotect | netfw | none |
+| pan_panos_globalprotect | pan:globalprotect | netfw | none |
```
Comment (ikheifets-splunk) [inline docs/sources/vendor/PaloaltoNetworks/panos.md:35]: Please, don't forget to also add [here](https://github.com/splunk/splunk-connect-for-syslog/blob/b915238328521657759b2748c37006e4e01e4616/docs/sources/vendor/PaloaltoNetworks/panos.md?plain=1#L21)

Comment (yaks77) [reply docs/sources/vendor/PaloaltoNetworks/panos.md:35]: done! ;)

**Review by ikheifets-splunk — APPROVED** [review]

**Pull request merged**